src/ai/backend/testutils/bootstrap.py

```diff
@@ -46,9 +46,9 @@ def etcd_container() -> Iterator[tuple[str, HostPortPair]]:
         capture_output=True,
     )
     container_id = proc.stdout.decode().strip()
-    container_info = wait_health_check(container_id)
-    host_port = int(container_info[0]['NetworkSettings']['Ports']['2379/tcp'][0]['HostPort'])
-    yield container_id, HostPortPair('127.0.0.1', host_port)
+    time.sleep(0.5)
```

Comment:
We can remove this `time.sleep()` call because `subprocess.run()` has already returned after termination of the `docker run -d` command where the container is already created and the container ID is printed to stdout, and the following line `wait_health_check()` will ensure the readiness of the container.